docs/datascience/notebooks-web.md

```diff
@@ -15,9 +15,14 @@ Visual Studio Code supports working with [Jupyter Notebooks](https://jupyter-not
 
 Using these browser-based platforms enables you to work on your notebooks (and beyond!) without having to install anything on your machine. You can read more about each of options and how to get started below:
 
-- [Jupyter notebooks on **GitHub Codespaces**](#jupyter-notebooks-on-codespaces)
-- [Remote tunneling with **VS Code Server** on VS Code for the Web](#remote-tunneling-with-vs-code-server)
-- [Connect to a remote **Jupyter server** on VS Code for the Web](#connect-to-a-remote-jupyter-server)
+- [Jupyter Notebooks on the web](#jupyter-notebooks-on-the-web)
+  - [Jupyter notebooks on Codespaces](#jupyter-notebooks-on-codespaces)
+  - [Jupyter notebooks on VS Code for the Web](#jupyter-notebooks-on-vs-code-for-the-web)
+  - [Remote tunneling with VS Code Server](#remote-tunneling-with-vs-code-server)
+  - [Connect to a remote Jupyter server](#connect-to-a-remote-jupyter-server)
+    - [Limitations](#limitations)
+  - [Continue working on Codespaces](#continue-working-on-codespaces)
+  - [Questions or feedback](#questions-or-feedback)
```

Comment:
I would stick with a curated link of sections, if you think it is helpful for readers, but not include the topic hierarchy.

Reply:
Changed back. [This extension](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) was trying to automatically create a "toc" for me..